### ISSUE TITLE
Chat server demo won't run in Python 3 because txsockjs appears not to be supported.

### ISSUE THREAD

**Issue by jMyles** [hendrix/hendrix#142]
See title.  Here's what happens when you try to install txsockjs on python 3:

```
Installing txsockjs…
Collecting txsockjs
  Downloading txsockjs-1.2.2.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-build-i22yxpor/txsockjs/setup.py", line 73
        except Exception, e:
                        ^
    SyntaxError: invalid syntax
```

**Comment by jMyles** [comment]
Perhaps txdarn will avail us: https://github.com/markrwilliams/txdarn

**Comment by jMyles** [comment]
Fixed in #143.
